pkg/combustion/templates/12-suma-register.sh.tpl

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+
+mkdir -p /etc/venv-salt-minion/
+
+{{ if .Get_SSL }}
```

Comment:
This brings up an interesting 3.1 discussion about if we want air-gapped as a definition parameter, which would let us have a validation step that tells the user if they enable something that isn't compatible with air-gapped.